### ISSUE TITLE
Authors don't show up on stories

### ISSUE THREAD

**Issue by mgax** [CRJI/theblacksea.eu#62]
- [ ] update the import script
- [ ] store it in a gist
- [ ] link the gist to this issue

**Comment by nightsh** [comment]
I ended up adding a new management command, it was easier to craft and since we might end up repeating it it only made sense to store it as one.

Small CLI sample before closing the issue:

```
$ ./manage.py fixturesauthors --help
usage: manage.py fixturesauthors [-h] [--version] [-v {0,1,2,3}]
                                 [--settings SETTINGS]
                                 [--pythonpath PYTHONPATH] [--traceback]
                                 [--no-color] [-n]
                                 fixture_path page_type

Imports all authors from the given Django fixture

positional arguments:
  fixture_path
  page_type             Page type (blog / story)
  -n, --no-commit

$ ./manage.py fixturesauthors blacktail/fixtures/theblacksea_04_stories.json story
INSERT INTO blacktail_story_authors (story_id, author_id) VALUES (1296, 38)
INSERT INTO blacktail_story_authors (story_id, author_id) VALUES (1297, 38)
INSERT INTO blacktail_story_authors (story_id, author_id) VALUES (1298, 38)
Skipping due to integrity error
INSERT INTO blacktail_story_authors (story_id, author_id) VALUES (1302, 7)
INSERT INTO blacktail_story_authors (story_id, author_id) VALUES (1302, 11)
INSERT INTO blacktail_story_authors (story_id, author_id) VALUES (1306, 51)
INSERT INTO blacktail_story_authors (story_id, author_id) VALUES (1306, 11)
```

**Comment by nightsh** [comment]
Also, we should be aware that no new articles have authors, since they are in free text format. While some of them might be identified by (normalized) name, some of them should be created and I felt this might be a better job for a human. There are just too many corner cases possible here and too few articles to craft a very smart script for it.

In [this example](https://theblacksea.eu/malta-files/article/en/cash-loans-czar-maltese-tax-escape-plan), the first two authors already exist and could be identified by name, but the other two should be created in the system. Also, watch out for diacritics (BLAŽ).

**Comment by nightsh** [comment]
To be fixed in #40.

**Comment by mgax** [comment]
Here are some examples of stories that have no authors:

https://wagtail.theblacksea.eu/stories/living-on-the-edge/
https://wagtail.theblacksea.eu/stories/secrets-of-the-international-criminal-court-jolie-clooney-and-the-world-fixer-psychosis/
https://wagtail.theblacksea.eu/stories/international-criminal-court-secrets-the-yezidi-lobby-scandal/
